engine/src/main/java/com/redhat/cloud/notifications/templates/TemplateService.java

```diff
@@ -51,6 +55,8 @@ void postConstruct() {
                 .addValueResolver(buildValueResolver(LocalDateTime.class, "toTimeAgo", LocalDateTimeExtension::toTimeAgo))
                 .addValueResolver(buildValueResolver(String.class, "toTimeAgo", LocalDateTimeExtension::toTimeAgo))
                 .addValueResolver(buildValueResolver(String.class, "fromIsoLocalDateTime", LocalDateTimeExtension::fromIsoLocalDateTime))
+                .addValueResolver(buildBiFunctionValueResolver(Context.class, "*", ActionExtension::getFromContext))
+                .addValueResolver(buildBiFunctionValueResolver(Payload.class, "*", ActionExtension::getFromPayload))
                 .addLocator(dbTemplateLocator)
```

Comment:
@gwenneg do you know if these resolvers are tested somewhere? 

I needed to create these because Context and Payload are no longer "Maps".

Reply:
Yes they are tested in DbQuteEngineTest:
https://github.com/RedHatInsights/notifications-backend/blob/5ca7e0316ba5b848b42417a68c682d48642b09e7/engine/src/test/java/com/redhat/cloud/notifications/templates/DbQuteEngineTest.java#L61-L68